chapter/8/big-data.md

```diff
@@ -1,11 +1,229 @@
 ---
 layout: page
 title:  "Large Scale Parallel Data Processing"
-by: "Joe Schmoe and Mary Jane"
+by: "Jingjing and Abhilash"
 ---
+## Introduction
+`JJ: Placeholder for introduction` The booming Internet has generated big data...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. {% cite Uniqueness --file big-data %}
 
-## References
+This chapter is organized in <label for="note1" class="margin-toggle sidenote-number"></label><input type="checkbox" id="note1" class="margin-toggle"/><span class="sidenote">JJ: need to fill in more stuff</span>
+
+- **Data paralleling**:
```

Comment:
Ok, this is getting better, but it still seems a bit confused. Maybe you've understood this now already from my other comments, by why not break things up differently? 

* Programming Models
  - Data parallelism (most popular, standard map/reduce/functional pipelining)
     - Limitations, iteration difficult due to the execution model of MapReduce/Hadoop
  - Graphs
  - Querying

* Execution Models
  - MapReduce (intermediate writes to disk)
    - Limitations, iteration, performance
  - Spark (all in memory)
    - Limitations ?

* Things people are building on top of MapReduce/Spark
  - FlumeJava? ...Etc
  - Ecosystem, everything interoperates with GFS or HDFS, or makes use of stuff like protocol buffers so systems like Pregel and MapReduce and even MillWheel...